src/tests/ftest/util/nvme_utils.py

```diff
@@ -0,0 +1,425 @@
+        if 'Write' in operation:
+            block_size = self.calculate_ior_block_size()
+            ior_cmd.block_size.update('{}'.format(block_size))
+            ior_cmd.flags.update(self.ior_flags)
+            #Store the container UUID and block size used for
+            #reading and verification purpose in future.
+            self.container_info["{}{}{}"
+                                .format(self.ior_daos_oclass,
+                                        self.ior_apis,
+                                        self.ior_transfer_size)] = [str(
+                                            uuid.uuid4()), block_size]
+        elif 'Read' in operation:
+            ior_cmd.flags.update(self.ior_read_flags)
+            #Retrieve the container UUID and block size for reading purpose
+            ior_cmd.block_size.update('{}'
+                                      .format(self.container_info
+                                              ["{}{}{}".format(
+                                                  self.ior_daos_oclass,
+                                                  self.ior_apis,
+                                                  self.ior_transfer_size)][1]))
+
+        # Define the job manager for the IOR command
+        manager = Mpirun(ior_cmd, mpitype="mpich")
+        manager.job.daos_cont.update(self.container_info
+                                     ["{}{}{}"
+                                      .format(self.ior_daos_oclass,
+                                              self.ior_apis,
+                                              self.ior_transfer_size)][0])
+        env = ior_cmd.get_default_env(str(manager))
+        manager.assign_hosts(self.hostlist_clients, self.workdir, None)
+        manager.assign_processes(self.processes)
+        manager.assign_environment(env, True)
+
+        # run IOR Command
+        try:
+            result = manager.run()
+            # Exception does not work so better to verify the proper
+            # completion too.
+            if 'Finished' not in result.stdout:
+                results.put("FAIL")
+        except CommandFailure as _error:
+            results.put("FAIL")
+
+    def calculate_ior_block_size(self):
+        """
+        Calculate IOR Block size to fill up the Server
+
+        Args:
+            None
+
+        Returns:
+            block_size(int): IOR Block size
+        """
+        #Check the replica for IOR object to calculate the correct block size.
+        _replica = re.findall(r'_(.+?)G', self.ior_daos_oclass)
+        if not _replica:
+            replica_server = 1
+        #This is for EC Parity
+        elif 'P' in _replica[0]:
+            replica_server = re.findall(r'\d+', _replica[0])[0]
+        else:
+            replica_server = _replica[0]
+
+        print('Replica Server = {}'.format(replica_server))
+        nvme_free_space = self.pool.get_pool_daos_space()["s_free"][1]
```

Comment:
It will good to a comment about about what conversion going on here. _tmp_block_size. Why divide by 100,etc?

Reply:
ok will add comment.